client/components/user-home/user-home.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import {connect} from 'react-redux'
 import {getEventsByStatus, fetchAllEvents} from '../../store/event'
-import {EventCard, EventActionButton} from '..'
+import {EventCard} from '..'
```

Comment:
??

Reply:
No clue. I just removed the Event Action Button Import